Source/WebCore/css/scripts/process-css-properties.py

```diff
@@ -3671,10 +3645,19 @@ def _generate_css_property_names_h_property_constants(self, *, to):
         to.write(f"constexpr auto firstShorthandProperty = {first_shorthand_property.id};")
         to.write(f"constexpr auto lastShorthandProperty = {last_shorthand_property.id};")
         to.write(f"constexpr uint16_t numCSSPropertyLonghands = firstShorthandProperty - firstCSSProperty;")
+        to.newline()
 
         to.write(f"extern const std::array<CSSPropertyID, {count_iterable(self.properties_and_descriptors.style_properties.all_computed)}> computedPropertyIDs;")
         to.newline()
 
+        to.write(f"template<CSSPropertyID C> struct PropertyNameConstant {{")
+        with to.indent():
+            to.write(f"static constexpr auto value = C;")
+            to.write(f"constexpr bool operator==(const PropertyNameConstant<C>&) const = default;")
```

Comment:
I think the `<C>` can be omitted here.

```suggestion
            to.write(f"constexpr bool operator==(const PropertyNameConstant&) const = default;")
```